src/main/java/edu/harvard/iq/dataverse/util/FileUtil.java

```diff
@@ -421,6 +433,11 @@ public static String retestIngestableFileType(File file, String fileType) {
     }
     
     public static String determineFileType(File f, String fileName) throws IOException{
+        final ResourceBundle bundle = BundleUtil.getResourceBundle("MimeTypeDetectionByFileName");
+        if (bundle.keySet().contains(fileName)) {
+            return bundle.getString(fileName);
+        }
+
```

Comment:
Same comment as below - this addition now makes the call on line 500 partially redundant unless that call drops checking for whole name again.